### PR TITLE
Feat(SPV-1149): fix error id not found for webhooks and paymails

### PR DESCRIPTION
**Pull request by dzolt-4chain**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: src/components/DataTable/DataTable.tsx [hunk 1]
```diff
@@ -33,15 +34,17 @@ interface DataTableProps<TData, TValue> {
   data: TData[];
   renderItem?: (row: Row<TData>) => React.ReactNode;
   renderInlineItem?: (row: Row<TData>) => React.ReactNode;
+  initialSorting?: ColumnSort[];
 }
 
-const initialSorting = { id: 'id', desc: false };
+const defaultInitialSorting: ColumnSort = { id: 'id', desc: false };
```
Comment (chris-4chain) [inline src/components/DataTable/DataTable.tsx:40]:
```suggestion
const defaultInitialSorting: ColumnSort[] = [{ id: 'id', desc: false }];
```

File: src/components/DataTable/DataTable.tsx [hunk 2]
```diff
@@ -50,7 +53,7 @@ export function DataTable<TData, TValue>({
     getPaginationRowModel: getPaginationRowModel(),
     getSortedRowModel: getSortedRowModel(),
     initialState: {
-      sorting: [initialSorting],
+      sorting: initialSorting ? initialSorting : [defaultInitialSorting],
```
Comment (chris-4chain) [inline src/components/DataTable/DataTable.tsx:56]:
```suggestion
      sorting: initialSorting ? initialSorting : defaultInitialSorting,
```

File: src/components/WebhooksTabContent/WebhooksTabContent.tsx [hunk 3]
```diff
@@ -17,6 +17,7 @@ export const WebhooksTabContent = ({ webhooks }: WebhooksTabContentProps) => {
           <DataTable
             columns={webhookColumns}
             data={webhooks}
+            initialSorting={[{ id: 'url', desc: false }]}
```
Comment (chris-4chain) [inline src/components/WebhooksTabContent/WebhooksTabContent.tsx:20]: Please make a constant for that:
```ts
const initialSorting = [{ id: 'url', desc: false }]
```

And then:
```suggestion
            initialSorting={initialSorting}
```

File: src/components/DataTable/DataTable.tsx (hunk 1 above)
Comment (dzolt-4chain) [reply src/components/DataTable/DataTable.tsx]: changed

File: src/components/DataTable/DataTable.tsx (hunk 2 above)
Comment (dzolt-4chain) [reply src/components/DataTable/DataTable.tsx]: changed

File: src/components/WebhooksTabContent/WebhooksTabContent.tsx (hunk 3 above)
Comment (dzolt-4chain) [reply src/components/WebhooksTabContent/WebhooksTabContent.tsx]: done

**Review by chris-4chain — APPROVED** [review]

**Review by mgosek-4chain — APPROVED** [review]

**Pull request merged**